micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmMemory.java

```diff
@@ -55,7 +57,9 @@ static boolean isAllocationPool(@Nullable String name) {
     }
 
     static boolean isLongLivedPool(@Nullable String name) {
-        return name != null && (name.endsWith("Old Gen") || name.endsWith("Tenured Gen") || "Shenandoah".equals(name)
+        return name != null && (name.endsWith("Old Gen") || name.endsWith("Tenured Gen") //
```

Comment:
```suggestion
        return name != null && (name.endsWith("Old Gen") || name.endsWith("Tenured Gen")
```

Reply:
The blank comment is there to prevent the formatter from causing weird formatting. It will move the Shenandoah condition up to this line and then break the comment up over multiple lines.